apps/meteor/tests/e2e/admin-feature-preview.spec.ts

```diff
@@ -0,0 +1,40 @@
+import { Users } from './fixtures/userStates';
```

Comment:
instead of creating an e2e test, I think creating few unit tests would be much more useful. also you didn test the whole point: if the default works for the users, you just tested if the page renders, which is expected already ;)